docs/basic_training/index.md

```diff
@@ -18,6 +18,18 @@ By the end of this course you should:
 4. Understand the different execution platforms supported by Nextflow
 5. Be introduced to the Nextflow community and ecosystem
 
+## Follow the training videos
+
+In our latest [nf-core training in March, 2023](https://nf-co.re/events/2023/training-march-2023), we used this training material to teach Nextflow and the recording is available for you to watch at your pace in multiple languages.
+
+Check the links below for the YouTube videos in the available languages:
+
+-   [English recording](https://youtube.com/playlist?list=PL3xpfTVZLcNhoWxHR0CS-7xzu5eRT8uHo)
+-   [Hindi recording](https://youtube.com/playlist?list=PL3xpfTVZLcNikun1FrSvtXW8ic32TciTJ)
+-   [Spanish recording](https://youtube.com/playlist?list=PL3xpfTVZLcNhSlCWVoa3GURacuLWeFc8O)
+-   [Portuguese recording](https://youtube.com/playlist?list=PL3xpfTVZLcNhi41yDYhyHitUhIcUHIbJg)
+-   [French recording](https://youtube.com/playlist?list=PL3xpfTVZLcNhiv9SjhoA1EDOXj9nzIqdS)
+
```

Comment:
I think it helps to embed at least one playlist, shows the video more clearly at a glance without reading any text.

Can of course show the playlists from other languages in the translations of this page.

```suggestion

<div style="text-align: center;">
    <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL3xpfTVZLcNhoWxHR0CS-7xzu5eRT8uHo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen="" data-ruffle-polyfilled=""></iframe>
</div>
```